pkg/storagedistribution/storagedistribution.go

```diff
@@ -111,10 +112,22 @@ func AddDisk(
 		}
 	}
 
+	driveRanges := make(map[string]string)
+	driveRangeStr := make([]string, 0)
+	for _, row := range dm.Rows {
```

Comment:
nit: Don't need to iterate twice. We can directly add the driveRangeStr slice, right?

Reply:
wanted to avoid duplicates incase 2 rows have the same ranges.

Reply:
Got it.